modules/plugins/snippets/luasnip/config.nix

```
@@ -9,12 +9,12 @@
 in {
   config = mkIf cfg.enable {
     vim = {
-      lazy.plugins = {
-        luasnip = {
-          package = "luasnip";
-          lazy = true;
-          after = cfg.loaders;
-        };
+      lazy.plugins.luasnip = {
+        package = "luasnip";
+        lazy = true;
+        after = cfg.loaders;
+        setupModule = "luasnip";
+        inherit (cfg) setupOpts;
```

```suggestion
        package = "luasnip";
        
        lazy = true;
        
        setupModule = "luasnip";
        inherit (cfg) setupOpts;
        
        after = cfg.loaders;
```
Sorry, but let's try to keep things consistent :b